src/raftstore/store/snap.rs

```diff
@@ -748,17 +758,13 @@ impl Snapshot for Snap {
             "snapshot" => %self.path(),
         );
         for cf_file in &self.cf_files {
-            delete_file_if_exist(&cf_file.clone_path).unwrap();
-            if self.hold_tmp_files {
-                delete_file_if_exist(&cf_file.tmp_path).unwrap();
-            }
-            if delete_file_if_exist(&cf_file.path).unwrap() {
+            if file_exists(&cf_file.path) {
```

Comment:
Please use `delete_file_if_exist` instead of `file_exists` because `delete` could be called in multi threads.